project_test/lib/pipeline_configurations.py

```diff
@@ -0,0 +1,88 @@
+from core_data_modules.cleaners import swahili, Codes
+
+from project_test.lib.test_schemes import CodeSchemes
+
+class CodingPlan(object):
+    def __init__ (self, raw_field, coded_field, coda_filename, cleaner=None, code_scheme=None, time_field=None,
+                    run_id_field=None, icr_filename=None, analysis_file_key=None, id_field=None):
+        self.raw_field = raw_field
+        self.coded_field = coded_field
+        self.coda.coda_filename = coda_filename
+        self.icr_filename = icr_filename
+        self.cleaner = cleaner
+        self.code_scheme = code_scheme
+        self.time_field = time_field
+        self.run_id_field = time_field
+        self.analysis_file_key = analysis_file_key
+
+        if id_field is None:
+            id_field = "{}_id".format(self.raw_field)
+        self.run_id_field = id_field
+
+class PipelineConfiguration(object):
+    
+    DEV_MODE = False
+
+    TEST_SHOWS_CODING_PLAN = [
+        CodingPlan(raw_field="internet_working_raw",
+        coded_field="internet_working_coded",
+        time_field="sent_on",
+        coda_filename="internet_working.json"
+        icr_filename="internet_working.csv",
+        run_id_field="internet_working_id",
+        analysis_file_key="internet_working_",
+        cleaner=None,
+        code_scheme=CodeSchemes.INTERNET),
+
+        CodingPlan(raw_field="water_filter_raw",
+        coded_field="water_filter_coded",
+        time_field="sent_on",
+        coda_filename="water_filter.json"
+        icr_filename="water_filter_icr.csv",
+        run_id_field="water_filter_id",
+        analysis_file_key="water_filter_",
+        cleaner=None,
+        code_scheme=CodeSchemes.WATER),
+    ]
+    
+    @staticmethod
+    def test_clean_age(text):
```

Comment:
Maybe call this `clean_age_within_range` or similar.

`test_clean_age` sounds too much like a unittest method.